### ISSUE TITLE
Expected input to metrics

### ISSUE THREAD

**Issue by Seilmast** [SFI-Visual-Intelligence/Collaborative-Coding-Exam#52]
I saw a argmax function was called over the logits before being passed into the metric wrapper. I've removed that, and am now passing the logits into this as they are needed for the entropy calculations. 
Please add a softmax / argmax to your individual metric classes.
